RELEASE_NOTES.rst

```diff
@@ -4,28 +4,28 @@ coala 0.10 - PolarBear
 ::
 
```

Comment:
@S2606 You should also change the message here:
* https://github.com/coala/coala/blob/master/coalib/collecting/Collectors.py#L141
* https://github.com/coala/coala/blob/master/coalib/misc/Constants.py#L24